tests/test_example_dags.py

```diff
@@ -29,6 +29,7 @@ def session():
 @pytest.mark.parametrize(
     "dag_id",
     [
+        "astro_homes_etl_dag",
```

Comment:
As part of #130 , I changed how we name the example DAGs. I suggest we use the same pattern. Also, it can help significantly debugging if the DAG filename matches the `dag_id` it creates.